### PR TITLE
Adding run method call for prompt-toolkit radiolist_dialog module, wh…

### LINKED ISSUE #37: Timezone Selection Prompt not Triggering when running PCA-Wizard
# 🐛 Summary #

In the latest gophish-tools image, when running pca-wizard "Assessment Name" the timezone selection GUI prompt is not triggered.

## To reproduce ##

1. Drop into COOL env (or build and tag image locally)
2. Try running docker run -it cisagov/gophish-tools pca-wizard "RV1234_test"

## Expected behavior ##

Timezone selection prompt should be triggered initially (See screenshot) and it is not.

## Any helpful log output or screenshots ##

Screen Shot 2021-07-21 at 3 49 40 PM

### PR DESCRIPTION
**Pull request by nickviola**
…ich is required in newer version

#  #

## 🗣 Description ##

Fixes the timezone selection prompt issue.

## 💭 Motivation and context ##

Resolves #37. The prompt was not displaying before due to a missing method call to .run() which is apparently required in the new version of the prompt_toolkit radiolist_dialog module

## 🧪 Testing ##

1. Drop into COOL env (or build and tag image locally)
2. Try running docker run -it cisagov/gophish-tools pca-wizard "RV1234_test" (the prompt should now trigger - see screenshot)

## 📷 Screenshots (if appropriate) ##

Screen Shot 2021-07-21 at 3 49 40 PM

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

**Review by jsf9k — APPROVED** [review]

**Comment by dav3r** [comment]
A couple of reminders:
* Please follow [these commit message guidelines](https://chris.beams.io/posts/git-commit/#seven-rules)- in this case, I noticed that you didn't write your commit message or this PR title the imperative mood.  It's not a huge deal, but just wanted to mention it since I noticed.
* Please make sure that you have done all of the items in the checklist, then check the boxes.  If any items don't apply to this PR (e.g. "All future TODOs are captured in issues, which are referenced in code comments." since you have no new TODOs) they can be deleted from the checklist.

**Review by dav3r — APPROVED** [review]
Good detective work here @nickviola 👍 

Also, this change should be versioned.  It can be part of the `v0.0.6` release that @JCantu248 is working on - please coordinate with him on that.  Feel free to reach out to me or anyone else on my team if you have questions about this.

**Comment by dav3r** [comment]
One other thing: please add "Resolves #37" to the "Motivation and context" section above.

**Comment by nickviola** [comment]
> One other thing: please add "Resolves #37" to the "Motivation and context" section above.

Updated!

File: tests/test_assessment_builder.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+from mock import patch
+
+# cisagov Libraries
+from assessment.builder import set_time_zone
+
+
+class TestSetTimezone:
+    """Test setting timezone logic."""
+
+    def test_radiolist_dialog_run_callsed(self):
```
Comment (dav3r) [inline tests/test_assessment_builder.py]: Typo?
```suggestion
    def test_radiolist_dialog_run_called(self):
```

**Pull request merged**